internal/pkg/manifest/job.go

```diff
@@ -192,3 +194,23 @@ func newDefaultScheduledJob() *ScheduledJob {
 		Environments: map[string]*ScheduledJobConfig{},
 	}
 }
+
+// ExposedPorts returns all the ports that are container ports available to receive traffic.
+func (cfg *ScheduledJob) ExposedPorts() ([]ExposedPort, error) {
```

Comment:
Huh can jobs expose any ports? do we allow that today I wouldn't have expected that